pages/category/index.vue

```diff
@@ -472,6 +619,10 @@ export default {
       flex: none;
       margin: 0 0 0 auto;
     }
+    @include for-mobile {
```

Comment:
default design should be for mobile-only, hence remove @include for-mobile

Reply:
Resolved